packages/website/components/meta-context.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+
+interface ContextInterface {
+  siteName: string;
+}
+
+const ContextMeta = React.createContext<ContextInterface>({ siteName: '' });
+
+export default ContextMeta;
+
+/* eslint-disable import/no-unresolved */
```

Comment:
if typescript also catches unresolved files we should just disable import/no-unresolved for .ts/.tsx files

Reply:
Since we have non-typescript files, we can't remove the rule, and eslint is rule everywhere or rule nowhere...

Reply:
not true, we already have ts/tsx overrides in our eslintrc